Version check on OpenShift

If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
I am trying to install Harbor on OpenShift 4.6.0-0.okd-2021-02-14-205305 which packages Kubernetes v1.19.2. However the [operator instructions](https://github.com/goharbor/harbor-operator/blob/master/docs/installation/installation.md#deploy-the-operator) fail with the error:

```
Release "harbor-operator" does not exist. Installing it now.
Error: chart requires kubeVersion: >= 1.16.0 which is incompatible with Kubernetes v1.19.2-1049+f173eb4a83e557-dirty
make: *** [Makefile:130: helm-install] Error 1
```

**Steps to reproduce the problem:**
```
git clone git@github.com:goharbor/harbor-operator.git
cd harbor-operator/
oc new-project harbor-operator
make helm-install
```

**Versions:**
Please specify the versions of following systems.

- harbor operator version: v2.2.0-rc3
- kubernetes version: v1.19.2

This also fails on MicroK8s 1.20

```
Release "harbor-operator" does not exist. Installing it now.
Error: chart requires kubeVersion: >= 1.16.0 which is incompatible with Kubernetes v1.20.2-34+350770ed07a558
make: *** [Makefile:130: helm-install] Error 1
```

--- a/charts/harbor-operator/Chart.yaml
+++ b/charts/harbor-operator/Chart.yaml
@@ -9,7 +9,7 @@ type: application
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 0.1.0
 
-kubeVersion: '>= 1.16.0'
+kubeVersion: '>= 1.16.0-0'
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to

